auth/init_secrets.go

```
@@ -0,0 +1,151 @@
+	CookieBlockKey            []byte
+}
+
+func writeSecrets(ctx context.Context, secrets SecretsSet, path string) error {
+	err := ioutil.WriteFile(filepath.Join(path, config.SecretNameClaimSymmetricKey), []byte(base64.RawStdEncoding.EncodeToString(secrets.TokenHashKey)), os.ModePerm)
+	if err != nil {
+		return fmt.Errorf("failed to persist token hash key. Error: %w", err)
+	}
+
+	logger.Infof(ctx, "wrote %v", config.SecretNameClaimSymmetricKey)
+
+	err = ioutil.WriteFile(filepath.Join(path, config.SecretNameCookieHashKey), []byte(base64.RawStdEncoding.EncodeToString(secrets.CookieHashKey)), os.ModePerm)
```

golangci-lint (1.64.8)

[medium] G306: Expect WriteFile permissions to be 0600 or less

(gosec)